frontend/src/app/components/UpdateList.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+import LayoutWrapper from './LayoutWrapper';
+import { newsUpdateL10nId } from '../lib/utils';
+
+export function prettyDate(date) {
+  return moment(date).format('MMMM Do, YYYY');
```

Comment:
Yeah, but I think that only applies to the [generic localeData API calls](https://momentjs.com/docs/#/i18n/locale-data/). This specific `format()` overrides whatever would come from a selected locale